pywbem/_subscription_manager.py

```diff
@@ -1347,7 +1420,8 @@ def _create_subscription(self, server_id, dest_path, filter_path, owned):
                     # It does not have any properties besides its keys,
                     # so checking the path is sufficient.
                     return sub_inst
-            sub_path = server.conn.CreateInstance(sub_inst)
+            sub_path = server.conn.CreateInstance(
+                sub_inst, namespace=server.interop_ns)
             sub_inst = server.conn.GetInstance(sub_path)
             self._owned_subscriptions[server_id].append(sub_inst)
         else:
```

Comment:
Two lines down is one more CreateInstance without namespace. I suggest adding it there for consistency.

Reply:
DONE